addon/services/raven.js

```diff
@@ -1,17 +1,12 @@
+import { assign } from '@ember/polyfills';
```

Comment:
Does this handle backwards-compatibility like the old code?

Reply:
this PR should only refactor to use RFC176 imports, but not change any behavior. can you adjust the code to preserve the old behavior?